### PR TITLE
Enable high availability for the controller-manger

### LINKED ISSUE #27: Create more than one pod of kubefed controller manager to enable high avilability
Now, we only start 1 kubefed controller manager pod, we can start more than 1 pod to enable high availability.

### PR DESCRIPTION
**Pull request by sohankunkerkar**
Fixes #27

### CODE REVIEW

File: deploy/resources/deployment.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ metadata:
     control-plane: controller-manager
     controller-tools.k8s.io: "1.0"
 spec:
-  replicas: 1
+  replicas: 3
```
Comment (marun) [inline deploy/resources/deployment.yaml]: Why 3?  2 should be sufficient.

Comment (sohankunkerkar) [reply deploy/resources/deployment.yaml]: @marun Alright I will change it to 2.

**Comment by sohankunkerkar** [comment]
@marun Ready for your review.

**Pull request merged**